src/scenes/CreateEvent/store.js

```diff
@@ -171,6 +177,32 @@ export default class CreateEventStore {
     );
   }
 
+  @action
+  onEnterDialog = async () => {
+    try {
+      const { data: { result } } = await axios.post(
+        Routes.api.transactionCost,
+        {
+          type: TransactionType.APPROVE_CREATE_EVENT,
+          token: Token.BOT,
+          amount: decimalToSatoshi(this.escrowAmount),
+          optionIdx: undefined,
+          topicAddress: undefined,
+          oracleAddress: undefined,
```

Comment:
Don't need to add any `undefined` keys, just leave them out. By default, every var not explicitly defined is `undefined`.